chromium_src/third_party/blink/public/mojom/worker/worker_content_settings_proxy.mojom

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) 2021 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+// This file contains patches to apply to an original mojom file.
```

Comment:
I think this comment is not necessary here. Looks like you copied it from an example file which is actually used in tests, I put the comment just to distinguish between initial and override files.

Reply:
Fixed in https://github.com/brave/brave-core/pull/11452/commits/49eab9cb56c965ad7ff26135418f6c855b6426dc